src/iop/channelmixerrgb.c

```diff
@@ -1918,12 +1918,12 @@ void gui_init(struct dt_iop_module_t *self)
                                                                               \
   g->normalize_##short = dt_bauhaus_toggle_from_params(self, "normalize_" #short);
 
-  NOTEBOOK_PAGE(red, R, N_("R"), N_("red"), FALSE)
-  NOTEBOOK_PAGE(green, G, N_("G"), N_("green"), FALSE)
-  NOTEBOOK_PAGE(blue, B, N_("B"), N_("blue"), FALSE)
-  NOTEBOOK_PAGE(saturation, sat, N_("colorfulness"), N_("colorfulness"), TRUE)
-  NOTEBOOK_PAGE(lightness, light, N_("brightness"), N_("brightness"), FALSE)
-  NOTEBOOK_PAGE(grey, grey, N_("grey"), N_("grey"), FALSE)
+  NOTEBOOK_PAGE(red, R, N_("R"), N_("red"), N_("red"), FALSE)
+  NOTEBOOK_PAGE(green, G, N_("G"), N_("green"), N_("green"), FALSE)
+  NOTEBOOK_PAGE(blue, B, N_("B"), N_("blue"), N_("blue"), FALSE)
+  NOTEBOOK_PAGE(saturation, sat, N_("colorfulness"), N_("colorfulness"), N_("colorfulness"), TRUE)
+  NOTEBOOK_PAGE(lightness, light, N_("brightness"), N_("brightness"), N_("brightness"), FALSE)
+  NOTEBOOK_PAGE(grey, grey, N_("grey"), N_("grey"), N_("grey"), FALSE)
```

Comment:
seems the tooltip and section values are always identical (not accidentally) so there's a little bit of unnecessary duplication here.

Reply:
Yes, but I was expecting the tooltip to be more meaningful at some point. So this code will make it easier to do that.